card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -240,6 +240,7 @@ class DefaultCardDelegate(
                         "- isReliable: ${detectedCardTypes.firstOrNull()?.isReliable}"
                 )
                 if (detectedCardTypes != outputData.detectedCardTypes) {
+                    inputData.selectedCardIndex = -1
```

Comment:
We have a little problem here... When you type the next digit the current selection will be deselected. This is because the the `DetectedCardType` from the output data has `isSelected = true` and the newly detected one has false

Reply:
Fixed 👍